src/systemcmds/tests/CMakeLists.txt

```diff
@@ -33,9 +33,7 @@
 
 set(srcs
 	test_adc.c
-	test_autodeclination.cpp
```

Comment:
If the .cpp files I added there are incorrect, I can remove them, but they caused unresolved symbols